lib/internal/symbols.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+module.exports = {
+  // alphabet symbols
+  A: 65, /*A*/
+  a: 97, /*a*/
+  Z: 90, /*Z*/
+  z: 122, /*z*/
+
+  // non-alphabetic symbols
+  DOT: 46, /*.*/
+  SLASH: 47, /*/*/
+  VERTICAL_BAR: 92, /*|*/
```

Comment:
92 should be `/*\*/` (backslash).

Reply:
@richardlau yeah, my bad, it's "backslash" of course